script/counter_inc_feePayer.ts

```diff
@@ -0,0 +1,97 @@
+import { deployments, ethers, config, getChainId } from "hardhat";
+import { Wallet as KlaytnWallet } from "@klaytn/ethers-ext";
+import { normalizeHardhatNetworkAccountsConfig } from "hardhat/internal/core/providers/util";
+
+async function getKlaytnSigners(): KlaytnWallet[] {
```

Comment:
How about moving this function to [hardhat-utils](https://github.com/klaytn/hardhat-utils) in the future PR?? If so, should we create a `util` directory there?  @kjeom @blukat29 

We'll just merge this PR for now, and remove this function in the future PR after hardhat-utils supports it.

Reply:
Oh, hardhat-utils is the place! I've been puzzled to find where to put this.
I strongly agree with your idea.

Reply:
hardhat-utils v0.0.7 has been released. Please update like https://github.com/klaytn/hardhat-utils/pull/10#issuecomment-1628583304 @nohkwak